nist_compliance/tasks/amazon_linux_2.yml

```diff
@@ -159,24 +159,24 @@
   tags: [ compliance, dbus ]
 
 - name: Ensure setgid and setuid are not set for /lib64/dbus file
-  file: 
-    path: /lib64/dbus-1/dbus-daemon-launch-helper 
+  file:
+    path: /lib64/dbus-1/dbus-daemon-launch-helper
     mode: 0750
   when: dbus_daemon.stat.exists == true
   tags: [ compliance, dbus ]
 
-- name: Remove ttyS0 from /etc/securetty
-  lineinfile: 
-    dest: /etc/securetty 
+- name: Remove anything but tty or vc/ from /etc/securetty
+  lineinfile:
+    dest: /etc/securetty
     state: absent
-    regexp: '^ttyS0$'
-  tags: [ compliance, tty ]
+    regexp: '^(?!tty[0-9]+|vc\/[0-9]+).*$'
```

Comment:
Sorry about all the whitespace changes. This is the key line to remove anything that is not tty[0-9] or vc/[0-9]